apkid/rules/apk/packers.yara

```diff
@@ -534,3 +530,21 @@ rule apkpacker : packer
     condition:
         is_apk and all of them
 }
+
+rule chornclickers : packer
+{
+
+  meta:
+    description = "Custom Chinese 'ChornClickers'"
```

Comment:
"Custom Chinese" is accurate but super redundant :-P. They're all "custom chinese". How about just "ChornClickers" with _maybe_ comment explaining the name was invented by us

Reply:
I dislike commenting too much the `description` tag. 

These are the two ways I've tried with:
```
[*] /media/edu/terita/apkid-todo/custom-chinese-packer-93/2ffc6fbe4e40417f35dbfb579c858d32
 |-> packer : ChornClickers

[*] /media/edu/terita/apkid-todo/custom-chinese-packer-93/2ffc6fbe4e40417f35dbfb579c858d32
 |-> packer : ChornClickers ('Ch'-china '-orn'-porn '-clickers')
```

Finally, I've decided to explain the origin in the source code like:
```
meta:
    description = "ChornClickers"
    explanation = "ChornClickers ('Ch'-china '-orn'-porn '-clickers')"
```